dev/run-tests.py

```diff
@@ -331,7 +331,6 @@ def run_scala_tests_maven(test_profiles):
 
 
 def run_scala_tests_sbt(test_modules, test_profiles):
-
```

Comment:
is such change generated by `dev/reformat-python` ?

Reply:
Yeah